docs/configuration/configuration.md

```diff
@@ -1294,6 +1294,11 @@ url: <string>
 write_relabel_configs:
   [ - <relabel_config> ... ]
 
+# Name of the remote write config, which if specified must be unique among remote write configs. 
+# The name will be used in metrics and logging in place of a generated value to help users distibguish between
```

Comment:
Distinguish has a typo in both descriptions.

Reply:
thanks :+1: